extras/PackageMaintenance.R

```diff
@@ -75,6 +75,7 @@ OhdsiRTools::formatRFile("R/AutoGeneratedDefinitions.R")
 
 
 ### test that code
+library(magrittr)
```

Comment:
dplyr subsumes magrittr, so if we depend on dplyr everywhere, we don't have to explicitly lists the magrittr dependency everywhere. 

In other words, I would just use `library(dplyr)` here.